index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <main id="main">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Hello friend. Good job so far 👏. There are some minor issues that need to be fixed. 
**The title**: It is important to change the title of your doc and make it more representative of your own project. It's more professional and can give a clue as to what readers should expect from your code.